### DISCUSSION TITLE
Recent version of GrandOrgue does not open Friesach's organ

### DISCUSSION THREAD

**Discussion by dubitans** [GrandOrgue/grandorgue#1623] [Help] [answered]
I am deeply grateful to the developers of GrandOrgue. He is a marvel of technology!
However, I have a problem with it: Friesach's organ cannot be opened in the recent version of GrandOrgue. It seems to be due to lack of memory. In this version, the «Memory limit (MB)» does not allow more than 1048576 . However, this organ is easily loaded in older versions.
Grateful for the attention.

**Comment by midimusic** [comment]
How much memory do you actually have in the computer? My experience is that Freisach should work OK with 16GB but will not work properly with 8GB. If you do have 16GB then it would be worth testing your menory because memory errors can produce all kinds of weird problems. 

csw900

**Reply by dubitans** [reply]
I've checked the memory of my computer, but the problem with the Friesach organ persists... Thanks anyway!

**Reply by alex-aotearoa** [reply, marked as answer]
Piotr Grabowski states that the 24 bit version needs 13 GB and the 16bit needs 6.6GB over and above what the computer is already using. In grand orgue>files> settings, you can select the "sample size" and limit the amount of memory available. My PC has now has 32GB of ram (after I couldnt get this one to run with 16GB, so I bought two new ram cards) and I still couldn't get the 24 bit version to run without issues. I then selected the 16 bit sample size and it ran with no apparent problems. I then tried limiting the memory limit to 8000 MB(8GB). This too, result in some issues such as unable to load some samples and then it would crash when I tried clicking some stops. Now I have changed the grand orgue settings memory limit to 12000 MB(12GB) and so far it is ok (only tried for a few minutes). So my conclusion is that you need more memory than what Piotr has prescribed. Please note after changing the settings you need to reload the organ set.

**Reply by dubitans** [reply]
Thank you very much for your kindness! Your answer is in line with my opinion.

**Comment by jason-barba** [comment]
See https://github.com/GrandOrgue/grandorgue/discussions/1752

//jens
